### DISCUSSION TITLE
[FEATURE REQ] JoininBox

### DISCUSSION THREAD

**Discussion by ostermayer** [minibolt-guide/minibolt#35] [Ideas]
Would be awesome to incorporate https://github.com/openoms/joininbox/blob/master/README.md#set-up-joininbox-on-linux into this project

**Comment by twofaktor** [comment]
What do you prefer, with the web interface or with the terminal interface? 

https://github.com/joinmarket-webui/jam

**Comment by ostermayer** [comment]
terminal interface is much more robust

**Comment by ostermayer** [comment]
also in the future jam and joininbox will not be mutually exclusive and
will coexist well but all joinmarket functionality currently exists with
joininbox while jam is fairly limited
